eng/common/scripts/Submit-PullRequest.ps1

```diff
@@ -124,5 +174,7 @@ else {
   # setting variable to reference the pull request by number
   Write-Host "##vso[task.setvariable variable=Submitted.PullRequest.Number]$($resp.number)"
 
-  AddLabels $resp.number $PRLabels
+  AddReviewers -prNumber $resp.number
```

Comment:
Can we do all these in the same github request instead of individual requests? The fewer requests the better for our rate limiting.

Reply:
We cant do them call because the URL for adding reviewers is different.

Reply:
Can we do either set of them at the same time we create the PR?

Reply:
That doesn't seem to be the case from the Docs https://developer.github.com/v3/pulls/#input